### PR TITLE
Remove default tokens

### PR DESCRIPTION
**Pull request by zhenik**

### CODE REVIEW

**Comment by zhenik** [comment]
Blocker https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/35

File: ansible/vault_consul_integration.yml [hunk 1]
```diff
@@ -18,19 +29,30 @@
 - name: Consul - create dev team policy
```
Comment (Neha-Sinha2305) [inline ansible/vault_consul_integration.yml:29]: - name: Consul - create admin team policy
```suggestion
- name: Consul - create dev team policy
```

File: ansible/templates/.env_default.j2 [hunk 2]
```diff
@@ -10,6 +10,4 @@ vault_pki=true
 
 # Default variables for hashistack
 VAULT_ADDR=http://127.0.0.1:8200
```
Comment (Neha-Sinha2305) [inline ansible/templates/.env_default.j2:12]: Should we also get rid of this?

Comment (zhenik) [reply ansible/templates/.env_default.j2:12]: That's good questions. @fredrikhgrelland WDYT?

Comment (fredrikhgrelland) [reply ansible/templates/.env_default.j2:12]: The vault addr we can keep. Vault defaults to https, so we must have that variable to use vault

**Review by Neha-Sinha2305 — APPROVED** [review]

File: ansible/templates/.env_default.j2 (hunk 2 above)
Comment (Neha-Sinha2305) [reply ansible/templates/.env_default.j2:12]: @fredrikhgrelland : Can't we explicitly set the VAULT_ADDR when and where needed in ansible files, similar to how we are setting VAULT_TOKEN and CONSUL_HTTP_TOKEN ?
As per my understanding, **VAULT_ADDR=http://127.0.0.1:8200** is box specific just like **VAULT_TOKEN=master** is, so I wonder why we should treat these two differently!

Comment (fredrikhgrelland) [reply ansible/templates/.env_default.j2:12]: Yes, we could, but I would argue this *is* different. the token we want you, as a user of the box or the module, to have an understanding of, but **where** vault is is a configuration setting that will be out of scope when using the module.

**Pull request merged**